docs/ref/validators.txt

```diff
@@ -285,3 +285,30 @@ to, or in lieu of custom ``field.clean()`` methods.
       ``decimal_places``.
     - ``'max_whole_digits'`` if the number of whole digits is larger than
       the difference between ``max_digits`` and ``decimal_places``.
+
+``FileExtensionValidator``
+--------------------------
+
+.. class:: FileExtensionValidator(allowed_extensions, message, code)
+
+    .. versionadded:: 1.10
+
+    Raises a :exc:`~django.core.exceptions.ValidationError` with a code of
+    ``'invalid_extension'`` if the ``value`` cannot be found in
+    ``allowed_extensions``.
+
+    .. warning::
+
+        Do not rely on validation of the file extension. Users can rename a
+        file to have any extension.
+
+``validate_image_file``
+-----------------------
+
+.. data:: validate_image_file
+
+    .. versionadded:: 1.10
+
+    Uses Pillow to ensure that the ``value`` is a valid image extension. The
+    full list of supported extensions can be found at the
```

Comment:
I would just link "a valid image extension" in the first sentence and omit the second one.